### PR TITLE
sv-SE: add new fields to swedish translation 6494-6596

### PR DESCRIPTION
**Pull request by Sadret**
Applying for issue(s):
- #2327
- #2328
- #2345
- #2354
- #2369
- #2378
- #2386
- #2387
- #2388
- #2414
- #2430
- #2446
- #2472
- #2473
- #2501
- #2509
- #2520
- #2527
- #2535
- #2550
- #2551
- #2562
- #2563
- #2572
- #2578
- #2594
- #2602
- #2609
- #2620
- #2625
- #2631
- #2641
- #2650
- #2658
- #2659
- #2678 
- #2679 
- #2692 

I also added some fields of #2509, #2527 and #2562, but did not handle all changed or added fields of these issues yet.
Lastly, I added some detail to STR 0582.

### CODE REVIEW

**Comment by Sadret** [comment]
Left to do:
#2509
#2539
#2558
changed fields

**Comment by Sadret** [comment]
Please squash.

**Review by tupaschoal — APPROVED** [review]
> I also added some fields of #2509, #2527 and #2562, but did not handle all changed or added fields of these issues yet.

If I understood correctly:
- #2509: missing the scenario descriptions part
- #2527 and #2562: Missing the updates to preexisting strings

Right? Seems fine to me to follow-up in another PR. I'll comment this in those 3 issues to make sure it's tracked and not check `sv-SE` there yet.

**Comment by tupaschoal** [comment]
Done, thanks for the extensive update to `sv-SE` @Sadret !

**Comment by Sadret** [comment]
> > I also added some fields of #2509, #2527 and #2562, but did not handle all changed or added fields of these issues yet.
> 
> If I understood correctly:
> 
> * [Strings 6530-6532 + Scenario descriptions #2509](https://github.com/OpenRCT2/Localisation/issues/2509): missing the scenario descriptions part
> * [String 6491, 6493, 6536 #2527](https://github.com/OpenRCT2/Localisation/issues/2527) and [Strings 6137-6138 + 6542-6543 #2562](https://github.com/OpenRCT2/Localisation/issues/2562): Missing the updates to preexisting strings
> 
> Right? Seems fine to me to follow-up in another PR. I'll comment this in those 3 issues to make sure it's tracked and not check `sv-SE` there yet.

Exactly. I will do this and the changed string issues in the next PR.

File: data/language/sv-SE.txt [hunk 1]
```diff
@@ -3597,6 +3601,109 @@ STR_6490    :Varning: Halvkompatibel parkversion
...
+STR_6534    :{WINDOW_COLOUR_2}Intensitetsfaktor: {BLACK}-{COMMA16} %
+STR_6535    :{WINDOW_COLOUR_2}Illamåendefaktor: {BLACK}-{COMMA16}%
+STR_6536    :Denna park sparades i en senare version av OpenRCT2. Parken sparades i v{INT32}, du använder för närvarande v{INT32}.
+STR_6537    :Tillåt normala gångvägar som köer
+STR_6538    :Visar normala gångvägar i kölistan i gångvägsfönstret
+STR_6539    :Bromsen stängd
+STR_6540    :{WINDOW_COLOUR_2}Särskilt tack till följande företag för att de tillåter oss att använda deras likhet:
+STR_6541    :{WINDOW_COLOUR_2}Rocky Mountain Construction Group, Josef Wiegand GmbH & Co. KG
+STR_6542    :Medverkande
+STR_6543    :Medverkande ...
```
Comment (Gymnasiast) [inline data/language/sv-SE.txt:3653]: This should use a proper ellipsis (…).

File: data/language/sv-SE.txt [hunk 2]
```diff
@@ -3597,6 +3601,109 @@ STR_6490    :Varning: Halvkompatibel parkversion
...
+STR_6576    :Lägger till speciella färger till färgpaletten
+STR_6577    :Blockbromshastighet
+STR_6578    :Ställ in hastighetsgräns för blockbromsar. I blockområdesläge kopplas intilliggande bromsar med en lägre hastighet till blockbromsen.
+STR_6579    :Blockbromsar ställs in på standardhastighet när de sparas som bandesign
+STR_6580    :Återställa
+STR_6581    :Är du säker på att du vill återställa alla kortkommandon på den här fliken?
+STR_6582    :Öppna fönstret för kortkommandon
+STR_6583    :{WINDOW_COLOUR_2}Bakvända tåg
+STR_6584    :Välj att köra tåg baklänges
+STR_6585    :Ändringar kan inte göras...
```
Comment (Gymnasiast) [inline data/language/sv-SE.txt:3695]: Idem

**Pull request merged**